src/Drivers/CMakeLists.txt

```diff
@@ -1,78 +1,214 @@
+include_directories(Dense)
+include_directories(MDS)
+include_directories(Sparse)
+include_directories(PriDec)
+include_directories(IpoptAdapter)
 
 install(
-  FILES IpoptAdapter.hpp
+  FILES IpoptAdapter/IpoptAdapter.hpp
   DESTINATION include
   )
 
 #add_executable(linalgTest.exe linalgTest.cpp)
 #target_link_libraries(linalgTest.exe hiop ${HIOP_MATH_LIBRARIES})
 
-add_executable(nlpDenseCons_ex1.exe nlpDenseCons_ex1.cpp nlpDenseCons_ex1_driver.cpp)
-target_link_libraries(nlpDenseCons_ex1.exe HiOp::HiOp)
+add_executable(nlpDenseCons_EX1.exe ${CMAKE_CURRENT_SOURCE_DIR}/Dense/nlpDenseCons_EX1.cpp  ${CMAKE_CURRENT_SOURCE_DIR}/Dense/nlpDenseCons_EX1_driver.cpp)
+target_link_libraries(nlpDenseCons_EX1.exe HiOp::HiOp)
 
-add_executable(nlpDenseCons_ex2.exe nlpDenseCons_ex2.cpp nlpDenseCons_ex2_driver.cpp)
-target_link_libraries(nlpDenseCons_ex2.exe HiOp::HiOp)
+add_executable(nlpDenseCons_EX2.exe  ${CMAKE_CURRENT_SOURCE_DIR}/Dense/nlpDenseCons_EX2.cpp  ${CMAKE_CURRENT_SOURCE_DIR}/Dense/nlpDenseCons_EX2_driver.cpp)
+target_link_libraries(nlpDenseCons_EX2.exe HiOp::HiOp)
 
-add_executable(nlpDenseCons_ex3.exe nlpDenseCons_ex3_driver.cpp)
-target_link_libraries(nlpDenseCons_ex3.exe HiOp::HiOp)
+add_executable(nlpDenseCons_EX3.exe  ${CMAKE_CURRENT_SOURCE_DIR}/Dense/nlpDenseCons_EX3_driver.cpp)
+target_link_libraries(nlpDenseCons_EX3.exe HiOp::HiOp)
```

Comment:
Perhap it would be better to have these in `CMakeLists.txt` file in `Dense` directory. The CMake file in `Drivers` directory should only include subdirectories.